recipes/dbus/dbus-glib.yaml

```diff
@@ -0,0 +1,54 @@
+inherit: [autotools, autoconf]
+
+metaEnvironment:
+   PKG_DESCRIPTION: "D-Bus client library (binding) for C using GLib"
+   PKG_LICENSE: "AFL-2.1 OR GPL-2.0-or-later"
+   PKG_VERSION: "0.112"
+
+depends:
+   - dbus::dbus-dev
+   - libs::expat-dev
+   - libs::glib-dev
+   - use: []
+     depends:
+      - dbus::dbus-tgt
+      - libs::expat-tgt
+      - libs::glib-tgt
+
+checkoutSCM:
+   scm: url
+   url: https://dbus.freedesktop.org/releases/dbus-glib/dbus-glib-${PKG_VERSION}.tar.gz
+   digestSHA1: 9eb81f50e36e349f57297d1a2a61b48707a551cc
+   stripComponents: 1
+
+multiPackage:
+   "":
+      depends:
+         - name: dbus::dbus-glib-tools
+           use: [tools]
+           tools:
+              target-toolchain: host-toolchain
```

Comment:
`host-compat-toolchain`. The `host-toolchain` does not fingerprint and might cause issues with binary packages between different distros.